src/ui/veles_mainwindow.cc

```diff
@@ -1114,9 +1128,27 @@ void VelesMainWindow::updateParsers(dbif::PInfoReply reply) {
   }
 }
 
+void VelesMainWindow::showDatabase() {
+  if (database_dock_widget_ == nullptr) {
+    createDb();
+  }
+
+  database_dock_widget_->raise();
```

Comment:
Hmm, one thing that might be slightly confusing about this implementation is that if database is in other main window (hidden under current one) user won't see any change after he click "Show database view" - although I don't know how hard is it to raise whole main windows in response.